.gitbook/0-about-panacea/1-data-exchange-protocol/1-DEP-problems-it-solves.md

```diff
@@ -0,0 +1,52 @@
+# DEP and the problems it solves
+
+DEP seeks to address the problems of decentralized data delivery that allows for data validation while ensuring data privacy:
+
+- [Data privacy](#data-privacy)
+- [Data validation](#data-validation)
+- [Incentive](#incentive)
+- [Decentralization](#decentralization)
+- [Scalability](#scalability)
+
+## Data privacy
+
+Since DEP is targeting sensitive data, the privacy of the data must be ensured throughout all data transfers. 
+No data must be exposed to anyone except promised data consumers during the data delivery.
+
+DEP uses data encryption in the whole data delivery process to ensure the privacy of sensitive data. 
+Symmetric and asymmetric key encryption is utilized to ensure that the privacy of data during all data delivery processes.
+
+## Data validation
+
+As data consumers, it is crucial to validate that the data they receive is correct.
+The data must be truthful, untampered, and conform to the requirements the consumer specified.
+
+DEP provides a way to verify data by utilizing oracle running in an secure enclave environment and DID-based verifiable presentation.
+
+## Incentive
+
+By providing data, the data provider can be guaranteed an incentive paid by the data consumer. 
+In this case, data delivery and payment must be processed atomically and simultaneously.
+
+DEP ensures this through Panacea blockchain transactions and data encryption.
```

Comment:
I wanted to talk about the problem that incentive payments and data delivery should be handled atomically. Our DEP is designed so that consumers can only see the data if the incentives are paid.

I think the confusion comes from the title `Incentive`. I'm not sure if `Atomic Incentive` would be better, so if you have any thoughts, please comment!

Reply:
Aha. It's not problem.
I understand it.